src/job/job.service.ts

```diff
@@ -0,0 +1,33 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { CreateClaimItemsDto } from './dto/create-claim-items.dto';
+
+@Injectable()
+export class JobService {
+  constructor(private readonly prismaService: PrismaService) {}
+
+  async createClaimItems(dto: CreateClaimItemsDto) {
+    return await this.prismaService.$transaction(async (tx) => {
+      const isExisting = await tx.job.findUnique({
+        where: { id: dto.id },
+      });
+      if (isExisting) throw new BadRequestException('Job already exists');
```

Comment:
(I don't have enough backend skills) I wonder what is the cheaper way between checking IDs like these lines or using table column constraints?

Reply:
I prefer to check duplicated value is existing rather than handling constraints error. In my thought, handling db error can wrap various case of error into single case: id duplicated. So I wanted to specific into single exact case.

Reply:
Okay. If you don't have an additional plan to work on this pull request, it seems good to merge now. 😁